diff --git a/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/decorators/CSSProps.kt b/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/decorators/CSSProps.kt
index 5aea5b3..bbd6003 100644
--- a/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/decorators/CSSProps.kt
+++ b/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/decorators/CSSProps.kt
@@ -143,7 +143,7 @@ inline fun <reified T : View> ViewDefinitionBuilder<T>.UseBoxShadowProp(crossinl
 
     val shadowStyle = mutableListOf<BoxShadow>()
     for (i in 0..<shadows.size()) {
-      val shadow = BoxShadow.parse(shadows.getMap(i), view.context) ?: continue
+      val shadow = BoxShadow.parse(shadows.getMap(i)) ?: continue
       shadowStyle.add((shadow))
     }
     body(view, shadowStyle)
